ci_framework/hooks/playbooks/fetch_compute_facts.yml

```diff
@@ -161,14 +161,14 @@
                                 {{ '{{' }} lookup("vars", networks_lower[network] ~ "_ip") {{ '}}' }}/{{ '{{' }} lookup('vars', networks_lower[network] ~ '_cidr') {{ '}}' }}
                             routes: {{ '{{' }} lookup('vars', networks_lower[network] ~ '_host_routes') }}
                         {{ '{%' }} endfor {{ '%}' }}
-                        {% endraw %}
+                        {% endraw -%}
```

Comment:
This change is currently what is causing the issue. By chomping all the whitespace from here to the end of the line, it eats the new line. So, this "op" is munged with the next one and the spacing is not right. oc kustomize just silently ignores this "op". You can see how the ending of it looks different from the others in:

https://logserver.rdoproject.org/98/398/c228d4f63078573a53f2a4ddc84a8c4441ec7b12/github-check/podified-multinode-edpm-deployment-crc/08d07b0/controller/ci-framework-data/artifacts/manifests/openstack/dataplane/cr/kustomization.yaml

Notice only one \n and then a bunch of whitespace after this op, when all the others look different. When I fixed this locally and ran oc kustomize, I then got networkConfig added.

Reply:
I pushed a new commit to hopefully address the whitespace issue. If it doesn't fix it, I suggest double checking that newtorkConfig.template got applied to the manifest being created for openstackdataplanenodeset.

Reply:
erk, that was tricky. Thanks @slagle ! I'll merge as-is since CI is all happy again :).